src/baseline_bots/bots/smart_order_accepter_bot.py

```diff
@@ -161,7 +176,8 @@ async def gen_proposal_reply(
                         )
                     )
                 messages.add_message(proposer, str(msg))
-                await self.send_message(proposer, str(msg))
+                if not (self.test_mode):
```

Comment:
what does test mode do? is this documented somewhere?

Reply:
Documented in ini